src/App.js

```diff
@@ -1,24 +1,29 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import States from "./Components/States";
+import { TextField, GridList, Typography } from "@material-ui/core";
```

Comment:
I think it'd be better to use standard h4 tags rather than the Typography component, esp since you're only using it in one spot right now

What would be even better (for HTML rules) is to use the h1 tag and then change the font size. Typically, if you have an h4 tag, there needs to be an h1, h2, and h3 tag somewhere on the page. Not necessary, but good practice.

Reply:
Nice, I never knew that.